drivers/rptun/rptun.c

```diff
@@ -280,6 +370,30 @@ static bool rptun_is_recursive(FAR struct rptun_priv_s *priv)
   return nxsched_gettid() == priv->tid;
 }
 
+static void rptun_command(struct rptun_priv_s *priv)
+{
+  FAR struct rptun_cmd_s *rptun_cmd = RPTUN_RSC2CMD(priv->rproc.rsc_table);
+  uint32_t cmd;
+
+  if (RPTUN_IS_MASTER(priv->dev))
+    {
+      cmd = rptun_cmd->cmd_slave;
+      rptun_cmd->cmd_slave = 0;
+    }
+  else
+    {
+      cmd = rptun_cmd->cmd_master;
+      rptun_cmd->cmd_master = 0;
+    }
+
+  switch (RPTUN_GET_CMD(cmd))
+    {
+      case RPTUN_CMD_PANIC:
+        PANIC();
+        break;
```

Comment:
Good to add explicit `default` case or change to `if`

Reply:
Done